### PR TITLE
WIP: Seeds DB with relevant data

### PR DESCRIPTION
**Pull request by Fatema-Moaiyadi**

### CODE REVIEW

File: handler/user_handler.go [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+
 	"github.com/vkhichar/asset-management/contract"
 	"github.com/vkhichar/asset-management/service"
-	"net/http"
 )
 
 func LoginHandler(userService service.UserService) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 
+		ctx := r.Context()
```
Comment (vkhichar) [inline handler/user_handler.go]: Remove this un-necessary code

File: server/insert_data.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"fmt"
+	_ "github.com/lib/pq"
```
Comment (vkhichar) [inline server/insert_data.go]: de we need this import statement?

File: service/user_service.go [hunk 3]
```diff
@@ -25,8 +26,8 @@ func NewUserService(repo repository.UserRepository, ts TokenService) UserService
 	}
 }
 
-func (service *userService) Login(ctx context.Context, email, password string) (*domain.User, string, error) {
-	user, err := service.userRepo.FindUser(ctx, email)
+func (s *userService) Login(ctx context.Context, email, password string) (*domain.User, string, error) {
```
Comment (vkhichar) [inline service/user_service.go]: remove this un-necessary code change

File: server/insert_data.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		return
+	}
+	repository.InitDB()
+
+	var d *sqlx.DB
+
+	d = repository.GetDB()
+
+	tx:=d.MustBegin()
+	tx.MustExec("Insert into users values ($1,$2,$3,$4)","Jan Doe","jandoe@gmail.com","12345",true)
```
Comment (vkhichar) [inline server/insert_data.go]: Specify column names in the query. this script will fail if we add more columns in the table

File: seed.sh [hunk 5]
```diff
@@ -0,0 +1,33 @@
+echo "Username: $1"
```
Comment (wsushmita) [inline seed.sh:1]: add command to run seed file in read me file

File: seed.sh [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+psql -U $1 $3 << EOF
+
+insert into users (name,email,password,is_admin) values ('Jan Doe','jandoe@gmail.com','12345',true);
+insert into users (name,email,password,is_admin) values ('Alisa Ray','alisaray@gmail.com','hello',false);
+insert into users (name,email,password,is_admin) values ('Tom Walters','tomwalters@gmail.com','tom123',false);
+insert into users (name,email,password,is_admin) values ('Alice Stephen','alicestep@gmail.com','alice776',true);
+
+insert into assets (id,status,category,purchase_at,purchase_cost,name,specifications) values ('ffb4b1a4-7bf5-11eb-9439-0242ac130002','active','Laptop','01/07/2017',50000,'Dell Latitude E5550','{"RAM":"4GB","HDD":"500GB","Generation":"i5"}');
+insert into assets (id,status,category,purchase_at,purchase_cost,name,specifications) values ('ffb4b488-7bf5-11eb-9439-0242ac130002','undermaintenance','Laptop','01/10/2017',40000,'IBM Thinkpad','{"RAM":"4GB","HDD":"500GB","Generation":"i5"}');
```
Comment (wsushmita) [inline seed.sh:13]: do we have undermaintenance status?

File: service/user_service.go [hunk 7]
```diff
@@ -3,6 +3,7 @@ package service
 import (
 	"context"
 	"errors"
+
```
Comment (wsushmita) [inline service/user_service.go:6]: remove this line

**Review by wsushmita — CHANGES_REQUESTED** [review]

**Comment by ranbharerp15** [comment]
Please make sure to add commit message related to changes, like "added bash script for SeedData", "added status param in Asset"

File: seed.sh (hunk 6 above)
Comment (Fatema-Moaiyadi) [reply seed.sh:13]: Yes we have defined status to take undermaintenance value

File: service/user_service.go (hunk 7 above)
Comment (Fatema-Moaiyadi) [reply service/user_service.go:6]: The editor is adding it while saving.

**Pull request merged**